mern/server/controller/flow.js

```diff
@@ -1,3 +1,6 @@
+// DATA ACCESS OBJECT - This file makes get calls to the MongoDB, 
```

Comment:
Just wondering if there is a reason the DAO is in the controller's folder?😅

Reply:
This has been resolved in Terry's recent clean architecture branch!